exec/RenderHtml.hs

```diff
@@ -91,6 +90,7 @@ pagesForm f =
     , f (Proxy :: Proxy PageUserSettings)
     , f (Proxy :: Proxy PageAdminSettingsEventsProtocol)
     , f (Proxy :: Proxy PageDelegateVote)
+--    , f (Proxy :: Proxy PageDelegationNetwork) -- FIXME
```

Comment:
I am happy if it works without any additional work. Previously it was a Page instance, not a FormPage.